about.html

```diff
@@ -2,406 +2,109 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>African Development Global Forum</title>
-		<link
-			href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
-			rel="stylesheet"
-			integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
-			crossorigin="anonymous"
-		/>
 		<link
 			href="./node_modules/@fortawesome/fontawesome-free/css/all.css"
 			rel="stylesheet"
 		/>
-
-		<link
-			rel="stylesheet"
-			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-		/>
 		<link rel="stylesheet" href="./assets/css/about.css" />
 	</head>
 	<body>
 		<div class="wraper">
 			<header>
-				<nav class="navbar navbar-collapse bg-light">
-					<div class="container-lg">
-						<a class="navbar-brand" href="index.html"
+				<nav class="menu-container">
+					<button class="menu-toggler" type="button">
+							<span class="menu-toggler-icon"><i class="fa fa-bars"></i></span>
+						</button>
```

Comment:
- [ ] I couldn't see the hamburger button in the mobile version, please 🙏🏽 fix it.

![image](https://user-images.githubusercontent.com/58642949/197136639-60d35efe-0416-4107-9631-a3fb32c2b483.png)